napari/layers/_tests/test_dask_layers.py

```diff
@@ -114,8 +85,8 @@ def test_dask_global_optimized_slicing(delayed_dask_stack, monkeypatch):
     v = ViewerModel()
     dask_stack = delayed_dask_stack['stack']
     layer = v.add_image(dask_stack)
-    # the first and the middle stack will be loaded
-    assert delayed_dask_stack['calls'] == 2
+    # the first and the middle stack will be loaded and for clim calculation 9 chunks will be accessed per stack
+    assert delayed_dask_stack['calls'] == 22
```

Comment:
```suggestion
    assert delayed_dask_stack['calls'] == MAX_NUMBER_OF_CHUNKS + 2
```

And for another asserts? Why this additional 2 calls?